sechub-test/src/test/java/com/mercedesbenz/sechub/test/report/ThymeLeafHTMLReportingTest.java

```diff
@@ -104,7 +111,7 @@ void example1_owasp_zap_sarif_report_is_transformed_to_expected_sechub_report_HT
         assertNotNull(htmlResult);
 
         assertTrue(htmlResult.contains(context.sechubJobUUID));
-        assertTrue(htmlResult.contains("XSS"), "The report must at least contain a cross site scripting vulnerability!");
+        assertTrue(htmlResult.contains("CWE-614"), "The report must at least contain the CWE-614 vulnerability!");
```

Comment:
You added your new test parts which is of course okay, but you removed old test parts which are IMO still valid. Please don't remove test parts which are still valid.

Please add
```
assertTrue(htmlResult.contains("XSS"), "The report must at least contain a cross site scripting vulnerability!");
``` 
again